client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -18,12 +18,22 @@ export const AppsViewLayout: React.FC = () => {
   const { data } = useAppsListing();
 
   const portalApp = findAppById(data, app.definition.id);
-
   const icon = portalApp?.icon || app.definition.notes.icon || 'Generic-App';
-  const userGuideLink =
-    portalApp?.userGuideLink || app.definition.notes.helpUrl;
 
   const htmlApp = data?.htmlDefinitions[appId];
+  const rawHtml = (htmlApp?.html as string) || '';
+
+  const userGuideLink =
+    htmlApp?.userGuideLink ||
+    portalApp?.userGuideLink ||
```

Comment:
Just checking, are the html app ids not in the data object, such that portalApp.userGuideLink would still work for html types?